superset/views/dashboard/api.py

```diff
@@ -80,32 +75,22 @@ def validate_slug_uniqueness(value):
 
 
 def validate_owners(value):
```

Comment:
This method is duplicated in  superset/views/chart/api.py - can we extract it to a shared util?

Reply:
Yap, planning to DRY things a bit more, still a work in progress

Reply:
Done